stream.py

```diff
@@ -1,23 +1,21 @@
 import base64
 import cv2
 import json
-import time
-from cache import redis_instance
 
 
 def stream_faces(ws):
     cascadePath = "cascades/haarcascade_frontalface_alt.xml"
     faceCascade = cv2.CascadeClassifier(cascadePath)
 
-    cap = cv2.VideoCapture(0)
```

Comment:
The VideoCapture method must be "0" by default because if I haven't a second webcam it will be raised an exception.